testplan/runners/pools/base.py

```diff
@@ -95,6 +98,7 @@ def __init__(self, **options) -> None:
         self.event_recorder = EventRecorder(
             name=str(self), event_type="Worker"
         )
+        self._discard_running = threading.Event()
```

Comment:
What is the difference of using threading.Event() v.s boolean.

And _discard_running v.s _discard_pending?

Reply:
in this simple use case i think they could be equivalent even in the GIL-free world, `threading.Event()` might only exist as a reminder

i tend to choose different variable names for the ease of grep